bundles/af-core/src/test/java/com/adobe/cq/forms/core/components/internal/models/v1/form/CheckBoxGroupImplTest.java

```diff
@@ -293,7 +294,44 @@ void testGetType() {
     @Test
     void testGetEnumNames() {
         CheckBoxGroup checkboxGroup = getCheckBoxGroupUnderTest(PATH_CHECKBOX_GROUP);
-        assertArrayEquals(new String[] { "m", "f", "o" }, checkboxGroup.getEnumNames());
```

Comment:
Can you keep the existing test case of getEnumNames since it is only deprecated, but needs to be tested ?

Reply:
okay